docker/testdata/test_grass_python.py

```diff
@@ -4,10 +4,10 @@
 
 # hint: do not use ~ as an alias for HOME
 with gs.setup.init(
-    # run in PERMANENT mapset of demolocation in GRASS GIS source
+    # run in PERMANENT mapset of demolocation in GRASSrce
```

Comment:
Typo

Reply:
```suggestion
    # run in PERMANENT mapset of demolocation in GRASS source
```

Yep, I saw that later on that vscode did that when I accidentally clicked two times before the first one finished  the UI lagged. I didn’t know it happened in the first batch too.